lib/utils/location.js

```diff
@@ -67,11 +67,20 @@ export const regexes: { [key: string]: RegExp } = {
 };
 /* eslint-enable key-spacing */
 
+export const execRegexes: { [k: string]: (path: string) => ?string[] } = {
+	comments: path => {
+		const match = regexes.comments.exec(path);
+		if (!match) return match;
+		match.splice(1, 2, match[1] || match[2] && match[2].replace(/^u.*\//, 'u_'));
+		return match;
+	},
+};
+
 export type PageType = 'wiki' | 'search' | 'profile' | 'stylesheet' | 'modqueue' | 'subredditAbout' | 'comments' | 'commentsLinklist' | 'liveThread' | 'inbox' | 'submit' | 'account' | 'prefs' | 'linklist';
 
 export const pageType = _.once((): PageType => {
 	const defaultPageType = 'linklist';
-	const pageTypes = ['wiki', 'search', 'profile', 'stylesheet', 'modqueue', 'subredditAbout', 'comments', 'commentsLinklist', 'liveThread', 'inbox', 'submit', 'account', 'prefs'];
+	const pageTypes = ['wiki', 'search', 'stylesheet', 'modqueue', 'subredditAbout', 'comments', 'commentsLinklist', 'profile', 'liveThread', 'inbox', 'submit', 'account', 'prefs'];
```

Comment:
```andytuba [7:17] i’m only a little worried about changing up the order of pagetype checks```

Yeah as long as no page types are added or removed it shouldn't really change anything (but we do rely heavily on the fallback to `linklist`)